morphir/runtime/test/jvm/src/org/finos/morphir/runtime/EvaluatorMDMTests.scala

```diff
@@ -1199,6 +1199,20 @@ object EvaluatorMDMTests extends MorphirBaseSpec {
         testEvalMultiple("diffInYears")("localDateTests", "diffInYearsTest", List(localDate, localDate.plusYears(2)))(
           Data.Int(2)
         ),
+        testEvalMultiple("diffInYearsAdjacent")(
+          "localDateTests",
+          "diffInYearsTest",
+          List(java.time.LocalDate.of(1900, 12, 31), java.time.LocalDate.of(1901, 1, 1))
+        )(
+          Data.Int(1)
+        ),
+        testEvalMultiple("diffInYearsFullYear")(
```

Comment:
Would rename this to `diffInYearsSameYear`

Reply:
fixed